examples/gcoap/Makefile

```diff
@@ -27,6 +27,22 @@ USEMODULE += shell
 USEMODULE += shell_commands
 USEMODULE += ps
 
+# DTLS configuration
+GCOAP_ENABLE_DTLS ?= 0
+ifeq (1,$(GCOAP_ENABLE_DTLS))
+	CFLAGS += -DCONFIG_GCOAP_ENABLE_DTLS=1
+	CFLAGS += -DCONFIG_GCOAP_DTLS_SESSIONS_MAX=1
+	# UDP Port to use (20220 is default for DTLS).
+	DTLS_PORT ?= 20220
+	CFLAGS += -DDTLS_DEFAULT_PORT=$(DTLS_PORT)
+
+	USEMODULE += gcoap_dtls
+	USEPKG += tinydtls
+	USEMODULE += tinydtls_sock_dtls
+	# tinydtls needs crypto secure PRNG
+	USEMODULE += prng_sha1prng
+endif
```

Comment:
Some of the config stuff might need to be moved lower to work with Kconfig. @leandrolanzieri @akshaim can one of you have a look?

Reply:
If those parameters are added also to Kconfig (which I would really recommend :) then yes, the configuration changes need to be moved bellow the inclusion of `Makefile.include`. That way one is able to check if Kconfig is being used.